src/modules/service-provider/repo/service-provider.entity.ts

```diff
@@ -31,4 +31,7 @@ export class ServiceProviderEntity extends TimestampedEntity {
 
     @Property({ nullable: true })
     public keycloakRole?: string;
+
+    @Property({ nullable: true })
+    public requires2fa?: boolean;
```

Comment:
I don't really like optional booleans, any reason why this has to be optional? The provider either requires 2fa or it doesn't

Reply:
I followed the rest of the properties - a lot of them are optional. But I agree that it does not really make sense here.

Fixed in [5596065](https://github.com/dBildungsplattform/dbildungs-iam-server/pull/648/commits/55960656bfa01f7dbb6cd2368742605f7e14b81c)